src/Components/Filter/ProductFilter.css

```diff
@@ -10,29 +10,27 @@
     background-color: var(--white-color);
     width: 0.3rem;
 }
-.side-bar::-webkit-scrollbar-thumb{
-    height: 0.5rem;
-    border-radius: 0.5rem;
-    background-color:var(--dark-gray-color);
-}
 .filter-heading{
-    margin-top: -1rem;
+    margin-top: 2rem;
```

Comment:
Here you can write margin: 2rem 2rem 0 0; instead of writing margin-top and margin-left seperately.